action/protocol/staking/handlers.go

```diff
@@ -426,6 +430,10 @@ func (p *Protocol) handleRestake(ctx context.Context, act *action.Restake, csm C
 	prevWeightedVotes := p.calculateVoteWeight(bucket, csm.ContainsSelfStakingBucket(act.BucketIndex()))
```

Comment:
move prevWeightdVotes to before using it

Reply:
To where? you mean before here?```candidate.SubVote(prevWeightedVotes) ```
we can't, bucket info will be changed